File: Python/cowsignal.py
import sys
sys.stdin = open("cowsignal.in", "r")
sys.stdout = open("cowsignal.out", "w")
dimensions = [int(x) for x in sys.stdin.readline().split()]
length, width, multiplier = dimensions[0], dimensions[1], dimensions[2]
original = []
modified = []
for i in range(0,length):
    original.append(sys.stdin.readline().strip())
for i in range(0, length):
    line = original[i]
    for e in range(width*multiplier):
        if e%width == 0 and not (i==0 and e==0):
            print('')
        print(line[e%width]*multiplier, end='')

File: CPP/cowsignal.cpp
#include <cstdio>
#include <vector>
#include <iostream>
using namespace std;

int main() {
    freopen("cowsignal.in", "r", stdin);
    freopen("cowsignal.out", "w", stdout);
    int m{}, n{}, k{};
    cin >> m;
    cin >> n;
    cin >> k;
    for (int i{}; i < m; i++) {
        string currentLine{};
        string multipliedLine{};
        cin >> currentLine;
        for(const char charactor : currentLine) {
            for (int j{}; j < k; j++) { multipliedLine += charactor; }
        }
        for (int j{}; j < k; j++) { cout << multipliedLine << endl; }
    }
}